### DISCUSSION TITLE
Does falcon support h2c (HTTP/2 over cleartext)?

### DISCUSSION THREAD

**Discussion by slondr** [socketry/falcon#192] [Q&A] [answered]
I am in the process of evaluating whether Falcon is a good fit for my application. One of the benefits of using Nginx proxying to Puma is that Nginx supports HTTP/2 over cleartext; does Falcon support this?

**Comment by ioquatix** [comment, marked as answer]
Yes, it does, but it requires extra configuration to force it on, because you don't have ALPN to select the protocol. It means you need to specify, for example, the port and the HTTP/2 protocol explicitly.
